students/alxsk/session07/html_render.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+"""
+A class-based system for rendering html.
+"""
+
+# This is the framework for the base class
+class Element(object):
+    open_tag = ""
+    close_tag = ""
+    content = ""
+
+    def __init__(self, content = None):
+        self.content = content '' #evaluating if something is true/false
+
+    def append(self, new_content):
+        try:
+            self.content = new_content
+        except: TypeError:
+            import pdb;pdb.set_trace()
+            print('ok')
```

Comment:
Nicely done.. it's better to import all your modules at the beginning of your script, that way if someone is trying to trace back what you are doing, they can just look at the top of your module to see where that particular imported method is coming from.